lib/scrabble.rb

```diff
@@ -1,18 +1,42 @@
 class Scrabble
+attr_reader :word
+
+  def initialize
+    @word = word
+  end
 
   def score(word)
-    1
+    if word.nil?
+      points = 0
+    else
+      chars = word.upcase.split('')
+
+      points = 0
```

Comment:
When you have each doing something to a collector object, it's a sign a better enumerable can be used.